core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -677,6 +679,7 @@ object KafkaConfig {
   val ConsumerGroupMaxHeartbeatIntervalMsDoc = "The maximum heartbeat interval for registered consumers."
   val ConsumerGroupMaxSizeDoc = "The maximum number of consumers that a single consumer group can accommodate."
   val ConsumerGroupAssignorsDoc = "The server side assignors as a list of full class names. The first one in the list is considered as the default assignor to be used in the case where the consumer does not specify an assignor."
+  val GroupConsumerUpgradePolicyDoc = "The config that enables the group protocol upgrade/downgrade. The valid values are " + Utils.join(Utils.enumOptions(classOf[GroupConsumerUpgradePolicy]), ",") + "."
```

Comment:
It would be great if we could extend the documentation here a little bit. I think that we need to call out that this is about converting classic group using the consumer embedded protocol to the consumer group protocol and vice versa. We could also call out the various policies with a small descriptions.

Reply:
nit: `ConsumerGroup...`